models/public/efficientnet-b0/efficientnet-b0.md

```diff
@@ -32,25 +32,25 @@ For details about this family of models, check out the [TensorFlow Cloud TPU rep
 
 ### Original Model
 
-Image, name - `image`,  shape - `[1x224x224x3]`, format is `[BxHxWxC]` where:
+Image, name - `image`,  shape - `[1x224x224x3]`, format is `[BxHxWxC]`, where:
 
 - `B` - batch size
 - `H` - height
 - `W` - width
 - `C` - channel
 
-Channel order is `RGB`.
+Channel order is `BGR`.
```

Comment:
That's not true: original - RGB, converted - BGR. It is applicable for the entire EfficientNet family and most of the public models.